### ISSUE TITLE
[Angular] auto-generated stories need to be cast to any

### ISSUE THREAD

**Issue by aloifolia** [storybookjs/storybook#11648]
**Describe the bug**
I just tried the upcoming Storybook (6.0.0-rc.13) in an Angular project. After initialisation I head to fix all stories because the `args` access expressions where invalid. 

**To Reproduce**
0. create an Angular project *in strict mode*
1. run `npx sb@next init`
2. run `npm run storybook`

**Expected behavior**
The standard storybook should be executable without manual type fixes.

**Screenshots**
<img width="583" alt="Screenshot 2020-07-22 at 16 26 42" src="https://user-images.githubusercontent.com/1295054/88188464-2dfcd280-cc38-11ea-8965-90f36a39178d.png">

**Code snippets**
I had to tweak the stories with any casts:

```ts
export const Primary = Template.bind({}) as any;
Primary.args = {
    primary: true,
    label: 'Button',
};
```
**System:**
Well... that was unexpected:
```
Environment Info:
(node:88453) UnhandledPromiseRejectionWarning: TypeError: e.filter is not a function
```

**Comment by shilman** [comment]
@gaetanmaisse @tooppaaa I guess we need to add typings to the TS templates... Can one of you take this?

**Comment by gaetanmaisse** [comment]
@aloifolia which version of Angular and TS are you using?

I guess Angular 6- as we have E2E tests for Angular 7, 8, 9, 10 and they are all :green:

**Comment by MiParekh** [comment]
Hi I think you added me by mistake please remove me

**Comment by aloifolia** [comment]
@gaetanmaisse Actually, I am using Angular 10.0.4.

I guess your E2E tests do not apply the strict mode for Angular. If you do not use strict mode (which entails stricter Typescript settings), you get inferences like these:

<img width="333" alt="Screenshot 2020-07-23 at 09 37 19" src="https://user-images.githubusercontent.com/1295054/88262397-664bf180-ccc8-11ea-9937-b3ad2b969e58.png">

So, the story objects are inferred as any and, consequently, `args` can be assigned without problems. 

In strict mode (`npx -p @angular/cli ng new projectname --strict`), however, the types are inferred more precisely:

<img width="390" alt="Screenshot 2020-07-23 at 09 41 13" src="https://user-images.githubusercontent.com/1295054/88262532-ab702380-ccc8-11ea-943d-d22f1a476635.png">


I just updated the first thread message to reflect this constraint.

**Comment by gaetanmaisse** [comment]
Hmm right! Didn't think about that while all the projects I work on are in strict mode 🤦
So we need to find a way to type these, but it can be harder than it looks because of all the addons 🤔

**Comment by Dongtun1977** [comment]
> @gaetanmaisse Actually, I am using Angular 10.0.4.
> 
> 
> 
> I guess your E2E tests do not apply the strict mode for Angular. If you do not use strict mode (which entails stricter Typescript settings), you get inferences like these:
> 
> 
> 
> <img width="333" alt="Screenshot 2020-07-23 at 09 37 19" src="https://user-images.githubusercontent.com/1295054/88262397-664bf180-ccc8-11ea-9937-b3ad2b969e58.png">
> 
> 
> 
> So, the story objects are inferred as any and, consequently, `args` can be assigned without problems. 
> 
> 
> 
> In strict mode (`npx -p @angular/cli ng new projectname --strict`), however, the types are inferred more precisely:
> 
> 
> 
> <img width="390" alt="Screenshot 2020-07-23 at 09 41 13" src="https://user-images.githubusercontent.com/1295054/88262532-ab702380-ccc8-11ea-943d-d22f1a476635.png">
> 
> 
> 
> 
> 
> I just updated the first thread message to reflect this constraint.

**Comment by shilman** [comment]
This was fixed in #11825 closing!
